extensions/blocks/instagram-gallery/edit.js

```diff
@@ -21,7 +21,7 @@ import {
 	withNotices,
 } from '@wordpress/components';
 import { useEffect, useState } from '@wordpress/element';
-import { __ } from '@wordpress/i18n';
+import { __, sprintf } from '@wordpress/i18n';
```

Comment:
Leaving a comment here but it's unrelated to the line.
There are two unused imports left: `Animate` and Lodash `times`.

Reply:
done.